documentation/user-interface/fixture-properties-dialog.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: Fixture Properties Dialog
+permalink: /documentation/user-interface/fixture-properties-dialog.html
+---
+# Fixture Properties Dialog
+
+The **Fixture Properties** dialog opens when you [create a new fixture](../guides/record-tests/README.md#create-fixtures) and allows you to specify its properties.
+
+The dialog also allows you to [modify](fixture-editor.md#modify-the-fixtures-properties) an existing fixture's properties.
+
+![Fixture Properties dialog](../../images/user-interface/dialogs/new-fixture-dialog.png)
+
+You can specify the following properties:
+
+* Fixture's name. When you enter a name, TestCafe automatically generates a file name and add it to the **File name** field.
+* File's name. You can use the generated name or specify your own.
+* Tested webpage's URL. TestCafe Studio supports HTTP and HTTPS protocols. You can also use the `file://` scheme to tests webpages in the local file system.
+* User credentials for webpages protected with HTTP [Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) or [Windows (NTLM)](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication) authentication. Expand the **HTTP Basic/NTLM authentication** section and specify a user name and password. For Windows authentication, TestCafe Studio also may need domain and workstation (PC) names. This information is automatically received from the machine on which TestCafe Studio is running.
```

Comment:
I'm not sure about `domain and workstation (PC) names`, because in TestCafe open source we allow to set them if they are required. Do we always receive these parameters automatically? Or we should add them in this dialog? @churkin

Reply:
We can get this info from the Windows machine, but not form the Mac or Linux. If client server requires this info then NTLM auth wont work on this platforms.

Reply:
https://github.com/DevExpress/testcafe-studio/issues/1715

Reply:
I've removed info about domain and workstation names